modules/post/multi/gather/jenkins_gather.rb

```diff
@@ -33,6 +33,7 @@ def initialize(info = {})
     )
     register_options(
       [
+        OptString.new('JENKINS_HOME', [ false, 'Set to the home directory of Jenkins. The Linux versions default to /var/lib/jenkins, but C:\\Windows\\System32\\config\\systemprofile\\AppData\\Local\\Jenkins.jenkins on Windows.', ]),
```

Comment:
I tried doing an install of Jenkins 2.401.1 on Windows Server 2022 and this folder doesn't exist from what I can tell?

Reply:
To be honest, I didn't test this, and found the Windows info on Google.

Reply:
Based on your installation, where does Jenkins get installed? That should be what value is by default.

Even if the "default" value is not correct, the previous method of "finding" the directory is used: https://github.com/rapid7/metasploit-framework/blob/0f5f495108036fefb89a6ebe04b5891c85b3d7a6/modules/post/multi/gather/jenkins_gather.rb#L340

Reply:
Hey @MegaManSec 👋 

I think [this](https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/Jenkins-Home-Directory-Location-Change-JENKINS_HOME-windows-linux-ubuntu) may have been the Windows info you came across on Google? I was looking into this and did a fresh install of Jenkins on a Windows VM via the installer. I had nothing at the default you provided:
```
C:\Windows\System32\config\systemprofile\AppData\Local\Jenkins\.jenkins
```
![image](https://github.com/rapid7/metasploit-framework/assets/69522014/584e7f83-4ac1-4c06-ac97-3296e3b070ae)

Which going by that link above is where Jenkins Home should be installed when using the installer, however my environment had it installed at:
```
C:\ProgramData\Jenkins\.jenkins
```
![image](https://github.com/rapid7/metasploit-framework/assets/69522014/e6bae48a-2824-4791-a593-3219c004c1ec)

I also, didn't specify any paths, went with the defaults the installer provided. May be worth looking at a few install methods mentioned for Windows in the above link to verify if there is multiple locations Jenkins Home can be installed or that links information is outdated and is maybe only found at `C:\ProgramData\Jenkins\.jenkins`.

Reply:
It should be OK now.

Reply:
Thanks, did you happen to test it, or just go off the testing I had completed above?

> May be worth looking at a few install methods mentioned for Windows in the above link to verify if there is multiple locations Jenkins Home can be installed or that links information is outdated and is maybe only found at `C:\ProgramData\Jenkins\.jenkins`.


Or have any thoughts on my suggestion?

Reply:
Unfortunately, I do not have a single Windows machine that I can use to test this, so I just went off your testing :)